src/modules/ctypes/freebsd/appindicator.jsm

```diff
@@ -0,0 +1,58 @@
+/* -*- Mode: js2; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
```

Comment:
Why not a single link:  modules/ctypes/freebsd -> modules/ctypes/linux ?
I'm wondering if we should not just switch and have modules/OS/ctypes (instead of modules/ctypes/OS) and keep a single link: module/freebsd -> module/linux...